src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -8,8 +8,14 @@
 
 ### Android AEPMessaging 2.0.1
 
+* Updates the AEPCore dependency to 2.0.1 which resolves inconsistencies between the Android and iOS MessagingDelegate classes.
+
+## February 3, 2023
+
+### Android AEPMessaging 2.0.0
+
 * Adds support for Adobe Journey Optimizer powered in-app messages.
-* Adds support for AEP 2.x extensions. Note, AEPMessaging 2.0.1 is not backwards compatible with the ACP 1.x extensions.
+* Adds support for AEP 2.x extensions. Note, AEPMessaging 2.0.0 is not backwards compatible with the ACP 1.x extensions.
```

Comment:
Major version update for [Adobe Journey Optimizer](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).

This release also adds the support for Adobe Journey Optimizer powered in-app messages.

Reply:
This update will make the release note consistent with other extension release notes.